pylintrc

```diff
@@ -216,7 +217,7 @@ single-line-if-stmt=no
 no-space-check=trailing-comma,dict-separator
 
 # Maximum number of lines in a module
-max-module-lines=1000
+max-module-lines=1100
```

Comment:
This seems an arbitrary increase. We should reason about this and some of the other changes rather than just forcing the changes to minimize our alerts.